composer/core/data_spec.py

```diff
@@ -214,15 +214,15 @@ class DataSpec:
 
     def __init__(
         self,
-        dataloader: Iterable,
+        dataloader: Union[Iterable, torch.utils.data.DataLoader],
```

Comment:
is `torch.utils.data.DataLoader` not a subclass of `Iterable` 😮 ? Was this always broken but only now getting caught by PyRight?

Reply:
I think so 😮